src/bin/casc.rs

```diff
@@ -35,3 +36,32 @@ fn main() -> std::io::Result<()> {
         Ok(s) => out_file.write_all(s.as_bytes()),
     }
 }
+
+// Create a list of policy files
+// Display info and proceed on errors.  As long as we have at least one file, errors should be
+// considered non-fatal
```

Comment:
> Why ignoring errors?

I think I was primarily thinking about the case where the errors aren't even .cas source files.  For example, if I have a directory tree owned by root in the same folder with my .cas source files, permission denied in that case should be non-fatal.  Maybe it would make sense to add some logic to say that if we have a path and the extension is .cas, then that becomes fatal?

> Moreover the case where there is no file is handled by the argument parser, not this helper.

Actually, the case I meant here is if there are files supplied but none of them are .cas files.  And it turns out, that case isn't actually handled currently (./target/debug/casc LICENSE returns 0 and doesn't generate an out.cil).  I'll get that fixed.

Reply:
I think errors should not be ignored by default. If really you have a use case where you have unreadable policy files, then a dedicated argument option should be provided to ignore such error explicitly.

If there is files explicitly provided as argument and without the "cas" extension (e.g. README.md or LICENSE), this should be treated as an error.

Reply:
The problem is that we're not just looking at explicitly provided files here.  We're recursively searching for all files.  If I run casc on my directory tree and have a README.md present, I want casc to silently ignore that regardless of whether there's an error with it.

I think I should modify this so that we pass up errors that either 1. aren't associated with a file or 2. are associated with a .cas file or a directory.  If we get an error from walkdir with a path to a non-cas file, I think it should be fine to skip it silently.